charts/kubernetes-agent/tests/tentacle-deployment_test.yaml

```diff
@@ -129,3 +129,11 @@ tests:
           value:
             limits:
               cpu: "200m"
+              
+  - it: "truncates long names"
+    set:
+      nameOverride: thisisaverylongstring34087236548972364802367y408923604872341234141
+    asserts:
+      - equal:
+          path: metadata.name
+          value: thisisaverylongstring34087236548972364802367y408923604872341234-tentacle
```

Comment:
@APErebus is the name meant to be shorter than 63 chars?

Reply:
it should automatically truncate it yes

Reply:
I should've been more specific - do we expect the whole name to be truncated to 63 chars? right now we truncate then add `-tentacle` to the name (which brings it over 63 chars)

Reply:
ooo, interesting

Reply:
This potential bug is covered by https://app.shortcut.com/octopusdeploy/story/78448/long-target-name-doesn-t-install